psydac/api/ast/linalg.py

```diff
@@ -457,7 +465,8 @@ def __hashable_new__(cls, ndim, comm_id=None, **kwargs):
 
         # Generate random tag, unique for all processes in MPI communicator
         tag = random_string(8)
-        if comm is not None and comm.size>1:
+        parallel = True
+        if comm is not None and comm.size>1 and parallel:
```

Comment:
See previous question about `parallel`

Reply:
it's deleted in [0a922eb](https://github.com/pyccel/psydac/pull/248/commits/0a922ebe5711d1e83d5f158155d0b4ca99315bae)